Makefile

```diff
@@ -0,0 +1,12 @@
+.PHONY: help all lint
+.DEFAULT_GOAL := all
```

Comment:
Minor, but I'd change DEFAULT_GOAL to help.

Reply:
Done!